### ISSUE TITLE
Add way to insert or replace a node's child text

### ISSUE THREAD

**Issue by dsherret** [dsherret/ts-simple-ast#63]
It would be useful to have the ability to insert or replace the text of a child. This operation would blow away all the child nodes of the node this is done on and users would not be able to insert or replace text outside the range of the node.

This would be useful because it would allow people to insert or replace text where there isn't an easy api implemented.

**Comment by dsherret** [comment]
Implemented in 0.56.0. Some examples:

```typescript
// insert text
sourceFile.insertText(0, "// some comment\n");
// replace text
sourceFile.replaceText([3, 7], "a"); // "// a comment\n"
// remove text
sourceFile.removeText([sourceFile.getPos(), sourceFile.getEnd()]);
```

These methods exist on SourceFiles, ClassDeclarations, EnumDeclarations, ConstructorDeclarations, GetAccessorDeclarations, MethodDeclarations, SetAccessorDeclarations, EnumDeclarations, FunctionDeclarations, InterfaceDeclarations, and NamespaceDeclarations.

Beware that if you use these methods, it will blow away any descendant nodes you might have a reference to (they will throw an error if you try to use them). It's very difficult to figure out where in the tree those previous descendants now are when using these methods so that's why that happens. You will just need to renavigate to those nodes to get a reference to them again.

**Comment by aleclarson** [comment]
Any way to append text?
